3604/0/about_strings.rb

```diff
@@ -0,0 +1,211 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+# rubocop:disable Metrics/ClassLength
+# :reek:TooManyMethods
+class AboutStrings < Neo::Koan
+  def test_double_quoted_strings_are_strings
+    string = 'Hello, World'
+    assert_equal true, string.is_a?(String)
+  end
+
+  def test_single_quoted_strings_are_also_strings
+    string = 'Goodbye, World'
+    assert_equal true, string.is_a?(String)
+  end
+
+  def test_use_single_quotes_to_create_string_with_double_quotes
+    string = 'He said, "Go Away."'
+    assert_equal 'He said, "Go Away."', string
+  end
+
+  def test_use_double_quotes_to_create_strings_with_single_quotes
+    string = "Don't"
+    assert_equal "Don't", string
+  end
+
+  def test_use_backslash_for_those_hard_cases
+    val_a = "He said, \"Don't\""
+    val_b = 'He said, "Don\'t"'
+    assert_equal true, val_a == val_b
+  end
+  # rubocop:disable Style/PercentLiteralDelimiters
+
+  def test_use_flexible_quoting_to_handle_really_hard_cases
+    val_a = %(flexible quotes can handle both ' and " characters)
+    val_b = %!flexible quotes can handle both ' and " characters!
+    val_c = %{flexible quotes can handle both ' and " characters}
+    assert_equal true, val_a == val_b
+    assert_equal true, val_a == val_c
+  end
+  # rubocop:enable Style/PercentLiteralDelimiters
+
+  def test_flexible_quotes_can_handle_multiple_lines
+    long_string = %(
+It was the best of times,
+It was the worst of times.
+)
+    assert_equal 54, long_string.length
+    assert_equal 3, long_string.lines.count
+    assert_equal "\n", long_string[0, 1]
+  end
+  # rubocop:disable Layout/IndentHeredoc
```

Comment:
Please let's fix it too

Reply:
When I fixed this error "IndentHeredoc" task isn't complete(Why? Because we should put white space before strings, so values ​​change in answers)